config.py

```diff
@@ -90,6 +94,10 @@ class Config(object):
     EMAIL_TOKEN_SALT = os.getenv('EMAIL_TOKEN_SALT', 'aardvark')
     # 24 hours in seconds
     EMAIL_TOKEN_EXPIRY = int(os.getenv('EMAIL_TOKEN_EXPIRY', '86400'))
+    # 72 hours in seconds
+    CREATE_ACCOUNT_EMAIL_TOKEN_EXPIRY = int(os.getenv('CREATE_ACCOUNT_EMAIL_TOKEN_EXPIRY', '259200'))
```

Comment:
Just thinking about it, is there a reason why we wanted 72hrs for the expiry of this token?  24 seems like long enough and means we don't have to add another config item

Reply:
The business case is that because the email is being requested by someone other than the person who'll actually click through the link, there'll be more of a lag for communicating the need to go set up their account etc.

We can chat about how long to set this with Mark etc if you'd like but there are already going to be unavoidable config changes as part of this card and one more isn't a huge issue

Reply:
No need to do this.